From c976b59eb23654d79e98b64beaaa92df43516aca Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Tue, 29 Mar 2016 13:30:50 -0400
Subject: [PATCH 081/347] ARM: dts: k2g-evm: update dts to enable mdio
 bindings

This patch comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

Update evm dts to enable mdio and phy for K2G EVM.

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
Signed-off-by: Sekhar Nori <nsekhar@ti.com>
(cherry picked from commit f728f9781589ac54d320a85afc6e28f71416b2bd)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/keystone-k2g-evm.dts |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/keystone-k2g-evm.dts b/arch/arm/boot/dts/keystone-k2g-evm.dts
index 692fcbb..92da02e 100644
--- a/arch/arm/boot/dts/keystone-k2g-evm.dts
+++ b/arch/arm/boot/dts/keystone-k2g-evm.dts
@@ -41,3 +41,15 @@
 	pinctrl-0 = <&uart0_pins>;
 	status = "okay";
 };
+
+&gbe0 {
+	phy-handle = <&ethphy0>;
+};
+
+&mdio {
+	status = "okay";
+	ethphy0: ethernet-phy@0 {
+		reg = <0>;
+		phy-mode = "rgmii-id";
+	};
+};
-- 
1.7.5.4

